src/app/components/task/task.component.ts

```diff
@@ -16,7 +16,7 @@ import {TaskType} from "../../../types/task.type";
   styleUrl: './task.component.scss'
 })
 export class TaskComponent implements OnInit, AfterContentChecked {
-  @Input() task!: TaskType;
+  @Input() task: TaskType = {} as TaskType;
```

Comment:
не приводи пустые объекты к конкретному типу
```suggestion
  @Input() task?: TaskType;
```